frameworks/kafka/tests/test_toggle_tls.py

```diff
@@ -0,0 +1,403 @@
+import logging
+import json
+import tempfile
+import pytest
+import uuid
+
+import sdk_cmd
+import sdk_install
+import sdk_marathon
+import sdk_plan
+import sdk_security
+import sdk_utils
+
+from security import transport_encryption
+
+from tests import auth
+from tests import config
+from tests import test_utils
+
+
+log = logging.getLogger(__name__)
+
+
+pytestmark = [
+    pytest.mark.skipif(sdk_utils.is_open_dcos(), reason="TLS tests required DC/OS EE"),
+]
+
+
+MESSAGES = []
+
+
+@pytest.fixture(scope='module')
+def service_account(configure_security):
+    """
+    Creates service account and yields the name.
+    """
+    try:
+        name = config.SERVICE_NAME
+        secret = "{}-secret".format(name)
+        sdk_security.create_service_account(
+            service_account_name=name, service_account_secret=secret)
+        # TODO(mh): Fine grained permissions needs to be addressed in DCOS-16475
+        sdk_cmd.run_cli(
+            "security org groups add_user superusers {name}".format(name=name))
+        yield {"name": name, "secret": secret}
+    finally:
+        sdk_security.delete_service_account(
+            service_account_name=name, service_account_secret=secret)
+
+
+@pytest.fixture(scope='module', autouse=True)
+def kafka_server(service_account):
```

Comment:
`kafka_server` and `kafka_client` look very similar at first glance to the one used in the other test.